tests/cli/test_cli_download.py

```diff
@@ -61,10 +73,30 @@ def test_download_tag_name_exclude_parent(self):
         self.parse_cli_string(cli_string)
         lightly.cli.download_cli(self.cfg)
```

Comment:
since every test has this line as well, it would call this multiple times, no?

Reply:
Yes, it called this once for every test
We could put those two lines into an extra function which is called directly with the cli_string.

Reply:
nope, I just realised its called twice thats all. If that is intended behaviour 1) to set it up and 2) to then call it correctly, then all is good